lib/mapper.js

```diff
@@ -30,7 +26,8 @@ exports.identify = function(msg){
 
 exports.track = function(msg){
   var ret = {};
-  if (msg.revenue()) ret.value = msg.revenue();
+  ret.properties = {};
+  if (msg.revenue()) ret.properties.value = Math.round(msg.revenue() * 100);
```

Comment:
Our internal conversion tracking expects the "value" on events to be sent in cents.